lib/index.js

```diff
@@ -24,7 +24,7 @@ const internals = {
 
         // Hoek
 
-        Hoek.Error
+        Hoek.AssertError
```

Comment:
This feels weak if we have multiple hoek versions in the dependency tree, but I guess it's no different from before.

Reply:
Exactly